Justfile

```diff
@@ -533,3 +533,26 @@ fzf-git:
     fi
   fi
   echo -e "$SUMMARY"
+
+#install jsonschema2pojo
+jsonschema2pojo:
```

Comment:
This needs a heavy refactor:
- versions are hardcoded, updatecli magic needed, although the versions on this one are _silly_  (`jsonschema2pojo-1.2.1`)
- probably should be yaml file based for other tools out there, could see `type` being added to `tools.yaml`

```yaml
jsonschema2pojo:
  type: java/directory
  repo: joelittlejohn/jsonschema2pojo
  version: jsonschema2pojo-1.2.1
  artifact: jsonschema2pojo-{version}.tar.gz
  contents: jsonschema2pojo-{version}/bin/:jsonschema2pojo
```

Type meaning its downloaded and extracted to share and contents are linked.

Reply:
Rather than undergoing the `tools.yaml` suggestion it probably makes sense to externalise the version first.

Reply:
I think that the standard updatecli might cope because conceptually this should work, since this configuration should generate you the right kind of updatecli manifest.

```
jsonschema2pojo:
  repo: joelittlejohn/jsonschema2pojo
  version: 1.2.1
  artifact: jsonschema2pojo-{version}.tar.gz
  updatecli:
    kind: regex
    pattern: ^jsonschema2pojo-[0-9]+\.[0-9]+\.[0-9]+$
    trimprefix: jsonschema2pojo-
```